js/os.ts

```diff
@@ -23,6 +23,25 @@ interface CodeInfo {
   sourceMap: string | undefined;
 }
 
+/** Check if running in terminal.
+ *
+ * isTTY(0); // stdin
+ * isTTY(1); // stdout
```

Comment:
Alternative API: isTTY().stdout: bool

Reply:
The other benefit of this API is you can completely cache the result, so the op is only ever calculated once.

Reply:
Yes I prefer the API @hayd suggested

Reply:
@hayd You say that it should return object?

Reply:
Yes, or an interface.